src/main/webapp/dashboard.html

```diff
@@ -36,6 +36,9 @@ <h4 id="user-name-location" class="card-title dark-emph"></h4>
           <a id="user-github" href="#" class="btn btn-primary" role="button">
             Github
           </a>
+          <a id="user-email" href="#" class="btn btn-primary" role="button">
+            Email Me
+          </a>
```

Comment:
I find it a bit confusing that you can email yourself. Is this an intended feature?

Reply:
I put this feature in place in case the dashboard becomes a breakout page for all mentors. That was the initial goal of the dashboard.

Reply:
Your point is valid though, I'll see if I can get the dashboard to function as a breakout page by the next iteration of the code, If I can't I'll remove it.